include/postprocessors/ElmIntegTotFissNtsPostprocessor.h

```diff
@@ -4,7 +4,12 @@
 #include "MooseVariableInterface.h"
 
 class ElmIntegTotFissNtsPostprocessor : public ElementIntegralPostprocessor
-/* public MooseVariableInterface */
+/**
+ * This class computes the postprocessor value for the total number of
+ * neutrons produced in one neutron generation from fission and delayed
+ * neutron precursors if specified.
+ * \f$ \sum_g \nu \Sigma_{g,f} \phi_g + \sum_i \lambda_i C_i \f$
```

Comment:
Is it possible to rename this class? That name is very hard to read and understand.

Reply:
Good point. I created issue #263 to track this. I won't rename it here because many input files use this postprocessor and editing all of them would bloat up this PR.